cmd/crane/cmd/mutate.go

```diff
@@ -78,6 +87,14 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 				return err
 			}
 
+			// set envvars if specified - split to a function as it is more
+			// complex then labels because in the config the key value pairs
+			// are just strings with a = in them
+			err = setEnvVars(&cfg.Config, envVars)
+			if err != nil {
+				return err
+			}
```

Comment:
nit:

```suggestion
			if err := setEnvVars(&cfg.Config, envVars); err != nil {
				return err
			}
```

(This makes it clearer that the `err` isn't going to escape this scope)